docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
If 1.7 would fail silently, and 1.8 loudly, it seems that the whole note ought to be under a `..versionchanged:` as with the flag itself, if a nod to it is given in the main documentation vs. the release notes.